README.md

```diff
@@ -1,121 +1,115 @@
-[![PyPI version](https://badge.fury.io/py/tslearn.svg)](https://badge.fury.io/py/tslearn)
-[![Documentation Status](https://readthedocs.org/projects/tslearn/badge/?version=latest)](http://tslearn.readthedocs.io/en/latest/?badge=latest)
-[![Build Status](https://travis-ci.org/rtavenar/tslearn.svg?branch=master)](https://travis-ci.org/rtavenar/tslearn)
-[![Codecov](https://codecov.io/gh/rtavenar/tslearn/branch/master/graph/badge.svg)](https://codecov.io/gh/rtavenar/tslearn)
-[![Downloads](https://pepy.tech/badge/tslearn)](https://pepy.tech/project/tslearn)
-
-`tslearn` is a Python package that provides machine learning tools for the analysis of time series.
-This package builds on `scikit-learn`, `numpy` and `scipy` libraries.
-
-If you would like to contribute to `tslearn`, please have a look at [our contribution guidelines](CONTRIBUTING.md).
-
-# Dependencies
-
-```
-Cython
-numpy
-numba
-scipy
-scikit-learn
-joblib
-numba
-```
-
-If you plan to use the `shapelets` module, `keras` and `tensorflow` should also be installed.
-`h5py` is required for reading or writing models using the hdf5 file format.
+<!-- Our title -->
+<div align="center">
+  <h3>tslearn </h3>
```

Comment:
This would ideally be larger, but Github puts a line under `<h1>` and `<h2>`. Even more ideally, this would be replaced with some logo :)

Reply:
It's already great like that. I agree that with a logo, it would be better, but we'll see that later :)